src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const sex = person.sex === 'f' ? 'husband' : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      <p className={`${person.age !== undefined ? 'Person__age' : 'Person__age--omit'}`}>
+        {`I am ${person.age}`}
+      </p>
```

Comment:
it's better to hide element from is there is not data to display
```suggestion
      (!!person.age && (
        <p className='Person__age'>
          {`I am ${person.age}`}
        </p>
      ))
```